Source/Core/VideoBackends/Vulkan/ObjectCache.cpp

```diff
@@ -123,8 +123,9 @@ bool ObjectCache::CreateDescriptorSetLayouts()
        VK_SHADER_STAGE_FRAGMENT_BIT},
   }};
 
-  static const std::array<VkDescriptorSetLayoutBinding, 1> standard_ssbo_bindings{{
+  static const std::array<VkDescriptorSetLayoutBinding, 2> standard_ssbo_bindings{{
       {0, VK_DESCRIPTOR_TYPE_STORAGE_BUFFER, 1, VK_SHADER_STAGE_FRAGMENT_BIT},
+      {1, VK_DESCRIPTOR_TYPE_STORAGE_BUFFER, 1, VK_SHADER_STAGE_VERTEX_BIT},
   }};
```

Comment:
Doesn't this mean this buffer is always accessible (or something like that), even on non-uber pipelines? `create_infos` always uses `static_cast<u32>(standard_ssbo_bindings.size()), standard_ssbo_bindings.data()`. Or is that handled by the `setLayoutCount--` in `CreatePipelineLayouts`?

Reply:
Yes, it's always accessible.  We use the same layout object for ubershaders and non-ubershaders, similar to how the GS buffer is included in the layout as long as GS is supported, regardless of whether the current pipeline uses GS or not.

I can remove it in the case where dynamic vertex loader is unsupported though (and probably should, since we don't bind anything there in that case).